### PR TITLE
releng: Add the contact information in case of job issue

### PR DESCRIPTION
**Pull request by cpanato**
To send an email notification to the release-team or
release-managers+alerts

Fixes: https://github.com/kubernetes/sig-release/issues/1216

/hold not 100% sure if the emails are correct to the jobs

/assign @alejandrox1 @hasheddan @saschagrunert @justaugustus

### CODE REVIEW

File: config/jobs/kubernetes/sig-release/release-branch-jobs/1.18.yaml [hunk 1]
```diff
@@ -721,6 +729,7 @@ postsubmits:
   kubernetes/kubernetes:
   - annotations:
       testgrid-dashboards: google-unit, sig-release-job-config-errors
+      testgrid-alert-email: release-managers+alerts@kubernetes.io
```
Comment (hasheddan) [inline config/jobs/kubernetes/sig-release/release-branch-jobs/1.18.yaml:732]: Do you think we should also add `release-team@kubernetes.io` to these bazel builds?

**Review by justaugustus — CHANGES_REQUESTED** [review]
/hold for discussion
cc: @kubernetes/release-engineering @kubernetes/release-team-leads 

Many moons ago, I created the job-config-errors dashboard --> https://testgrid.k8s.io/sig-release-job-config-errors
There was a little handwavyness about what classifies as a "job config error", but high on that list is:

"This job is on a SIG Release dashboard, but does not have a alert email for the owning SIG"

On the whole, I'm not excited about putting us on the hook for owning alerts for jobs we're **_NOT_** ultimately responsible for remediating.

So two things I'd like to see:
- Outreach to the relevant SIGs to get an appropriate email address for alerting on their jobs
- A test to report when jobs in `release-branch-jobs/` do not have a SIG/component area email address to alert on

File: config/jobs/kubernetes/sig-release/release-branch-jobs/1.18.yaml (hunk 1 above)
Comment (cpanato) [reply config/jobs/kubernetes/sig-release/release-branch-jobs/1.18.yaml:732]: we can add, I don't have a strong opinion on that

**Comment by cpanato** [comment]
@justaugustus makes sense, this was a conversation we have back in time with @alejandrox1 but i will check each job and talk with the appropriate sigs
thanks for your feedback

**Comment by hasheddan** [comment]
> A test to report when jobs in release-branch-jobs/ do not have a SIG/component area email address to alert on

I can get this added :+1:

**Comment by justaugustus** [comment]
SGTM!
p.s. some of what you need is already written. Take a look around here: https://github.com/kubernetes/test-infra/blob/65eab84d72637b05f50d0e3640273e2b99347851/config/tests/jobs/jobs_test.go#L986

**Comment by cpanato** [comment]
will close this PR for now
/close

**Pull request closed without merging**